workers/unity/Packages/com.improbable.gdk.core/Config/ConnectionConfig.cs

```diff
@@ -29,6 +67,14 @@ public ConnectionParameters CreateConnectionParameters()
             return connectionParameters;
         }
 
+        /// <summary>
+        ///     Checks that a string value is non-null or empty.
```

Comment:
I can't tell what this function does from this description. From this I would think it's literally String.IsNullOrEmpty, which raises the question of why would I use it.

Reply:
That is literally what it does